### PR TITLE
Make User Feedback public

### PR DESCRIPTION
**Pull request by dcramer**
Remove the feature switch our user reports/feedback.

/cc @getsentry/team

### CODE REVIEW

**Comment by mattrobenolt** [comment]
:cookie:

**Comment by JTCunning** [comment]
LGTM

**Comment by ckj** [comment]
I realize that you're trying to get this out asap, but a couple thoughts: 

1. we should put the message inside the user box on the issue detail page

2. the modal UI itself could use some love. we've (or maybe just i've) been looking at it as a first draft. feels like we should polish it before going full on with it.

/2 cents

File: src/sentry/api/serializers/models/project.py [hunk 1]
```diff
@@ -60,7 +60,11 @@ def serialize(self, obj, attrs, user):
         from sentry import features
 
         feature_list = []
+<<<<<<< 596af09c245b4338d89ecdd5244a0df42ba4253a
```
Comment (ckj) [inline src/sentry/api/serializers/models/project.py]: Did you mean to do this?

Comment (mattrobenolt) [inline src/sentry/api/serializers/models/project.py]: lol

**Pull request merged**